_posts/2015-11-05-Les-Gares-de-Paris.md

```diff
@@ -0,0 +1,19 @@
+# Les Gares de Paris
+
+Il y a six gares principales à Paris : Gare de Lyon, Gare du Nord, Gare de l'Est, Gare Saint Lazare, Gare d'Austerlitz, et Gare Montparnasse. Les gares sont toutes jolies, et chaque gare a sa région ! Pratique, non ? D'ailleurs elles sont toutes mieux que Penn Station. Il est utile de se familiariser avec ces gares.
```

Comment:
"D'ailleurs, "

Maybe mentioning where is Penn Station. Not everyone knows NYC 😉